src/External/GLFW/CMakeLists.txt

```diff
@@ -375,7 +375,7 @@ endif()
 #--------------------------------------------------------------------
 # MODIFICATION BY Open3D: overwrite some GLFW cmake variables
 #--------------------------------------------------------------------
-set(GLFW_INCLUDE_DIRS "${Open3D_SOURCE_DIR}/External/GLFW/include")
+set(GLFW_INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}/include")
 set(GLFW_LIBRARIES "glfw")
 set(GLFW_INCLUDE_DIRS ${GLFW_INCLUDE_DIRS} PARENT_SCOPE)
 set(GLFW_LIBRARIES ${GLFW_LIBRARIES} PARENT_SCOPE)
```

Comment:
At the end of this CMakeLists file, there should be install rules as in other CMakeLists files, something like:
```
+# install
+install(TARGETS ${GLEW_LIBRARY}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
+        LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
```

In its current status, if ``BUILD_GLFW`` is on, and make install, GLFW won't be install into the Open3D install directory.